### PR TITLE
Roe 538- Add empty trust field validation

### PR DESCRIPTION
**Pull request by dalebradley**
### JIRA link
https://companieshouse.atlassian.net/browse/ROE-538

### Change description
Adds validation for when the trust details input box is empty

### Work checklist

- [ x] Tests added where applicable
- [x ] UI changes meet accessibility criteria

### Merge instructions

We are committed to keeping commit history clean, consistent and linear. To achieve this, this commit should be structured as follows:

```
<type>[optional scope]: <description>
```

and contain the following structural elements:

- fix: a commit that patches a bug in your codebase (this correlates with PATCH in semantic versioning),
- feat: a commit that introduces a new feature to the codebase (this correlates with MINOR in semantic versioning),
- BREAKING CHANGE: a commit that has a footer `BREAKING CHANGE:` introduces a breaking API change (correlating with MAJOR in semantic versioning). A BREAKING CHANGE can be part of commits of any type,
- types other than `fix:` and `feat:` are allowed, for example `build:`, `chore:`, `ci:`, `docs:`, `style:`, `refactor:`, `perf:`, `test:`, and others,
- footers other than `BREAKING CHANGE: <description>` may be provided.

### CODE REVIEW

File: src/validation/error.messages.ts [hunk 1]
```diff
@@ -97,5 +97,7 @@ export enum ErrorMessages {
   OCCUPATION_INVALID_CHARACTERS = "Occupation must only include letters a to z, numbers, and special characters such as hyphens, spaces and apostrophes",
   ROLES_AND_RESPONSIBILITIES_INVALID_CHARACTERS = "Role and responsibilities must only include letters a to z, numbers, and special characters such as hyphens, spaces and apostrophes",
   CONTACT_NAME_INVALID_CHARACTERS = "Contact name must only include letters a to z, numbers, and special characters such as hyphens, spaces and apostrophes",
-  EMAIL_INVALID_FORMAT = "Enter an email address in the correct format, like name@example.com"
+  EMAIL_INVALID_FORMAT = "Enter an email address in the correct format, like name@example.com",
+  // Empty fields
+  TRUST_DATA_EMPTY = "Paste the trust information from the Excel document into the box."
```
Comment (mwestacott) [inline src/validation/error.messages.ts:102]: Empty field check texts are at the top of this file (although admittedly not commented, so not clear). Suggest moving this up to just under line 31 though, just to keep them all together.

File: test/controllers/trust.information.controller.spec.ts [hunk 2]
```diff
@@ -73,13 +72,5 @@ describe("TRUST INFORMATION controller", () => {
       expect(resp.status).toEqual(302);
       expect(resp.header.location).toEqual(config.TRUST_INFO_URL);
     });
-
-    test("catch error when rendering the page", async () => {
```
Comment (mwestacott) [inline test/controllers/trust.information.controller.spec.ts:77]: Wondering why this test has been removed - surely it's still a valid test?

You need to add a test for when trust data box is empty though now, to check the new validation implementation you've put in place. Plenty of examples of doing that in other controllers.

Comment (mouhajer-ch) [reply test/controllers/trust.information.controller.spec.ts:77]: Probably something like this will do
```
const resp = await request(app)
   .post(config.TRUST_INFO_URL);
   .send({TrustKey: "Something" });
```

Comment (dalebradley) [reply test/controllers/trust.information.controller.spec.ts:77]: Yep, I removed it as it wasn't testing the controller code. After adding validation tests, I've managed to get it working again (thanks Mo!)

**Review by mouhajer-ch — APPROVED** [review]

File: src/validation/error.messages.ts [hunk 3]
```diff
@@ -29,6 +29,7 @@ export enum ErrorMessages {
   COUNTRY = "Select a country from the list",
   UK_COUNTRY = "Select a country",
   POSTCODE = "Enter a postcode",
+  TRUST_DATA_EMPTY = "Paste the trust information from the Excel document into the box.",
```
Comment (mwestacott) [inline src/validation/error.messages.ts:32]: No full-stops at the end of any of our other validation error messages so you may want to check this with an analyst. Up to you.

**Review by mwestacott — APPROVED** [review]
Tests looking good. Just that one minor comment. Approving - up to you.

File: src/validation/error.messages.ts (hunk 3 above)
Comment (dalebradley) [reply src/validation/error.messages.ts:32]: Fair point. Removed after speaking to our analyst
